extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml

```diff
@@ -109,5 +129,11 @@
         <description>Allows to control the color temperature of light.</description>
         <category>ColorLight</category>
     </channel-type>
-
+    
+    <channel-type id="group_state">
```

Comment:
If there are constellations which don't expose a dimmer (5851) but only an ON/OFF (5850) so that this is really needed, I would suggest keeping it group-agnostic, i.e. name it `switch` and remove all the references to groups from label & description, so that it can be used also directly in case they ever decide to offer a non-dimmable lamp.